caddy-mixin/dashboards/caddy-overview.json

```diff
@@ -685,6 +622,6 @@
   },
   "timezone": "",
   "title": "Caddy Overview",
-  "uid": "9B0qPnfMz",
-  "version": 8
+  "uid": "Al2uDkD7z",
```

Comment:
Hmmn.. I'm not sure what the implication of changing the UID might be. I know in some cases it's used to link from other dashboards, so if a customer is expecting to link to Caddy with it's current UID this will break it.

Could we restore the old UID, and set the version to 9?